ci/test.sh

```diff
@@ -39,6 +39,11 @@ done
 ./tools/install_config.sh --config=config/config_dev.json --update
 ./tools/setup_local_database.sh --nopassword --db_user ubuntu --db_name circle_test
 
+# Verify that data generation works.
+./tools/generate_fake_data.sh \
```

Comment:
Does this actually abort if generation fails?

Reply:
Server 500 -> client error -> exit status != 0, so yep.